opam

```diff
@@ -36,3 +36,6 @@ OPAM packages and the host package management system. It can query OPAM for the
 right external dependencies on a set of packages, depending on the host OS, and
 call the OS's package manager in the appropriate way to install them.
 """
+post-messages: [
+  "opam-depext is deprecated when used with opam >= 2.1. Please use opam install directly instead" {opam-version >= "2.1"}
```

Comment:
```suggestion
  "opam-depext is unnecessary when used with opam >= 2.1. Please use opam install directly instead" {opam-version >= "2.1"}
```
The new support is for backwards compatibility rather than for deprecation - the plugin doesn't display a warning either if you run with `OPAMCLI=2.0 opam depext ...`